release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -499,7 +507,7 @@ The following OpenShift CLI (`oc`) commands were removed with this release:
 * `oc adm migrate storage`
 
 [id="ocp-4-11-custom-scheduler"]
-==== Support for deploying a custom scheduler manually removed
+==== Support for deploying a custom scheduler manually has been removed
```

Comment:
The preview is still showing the unchanged version.<img width="776" alt="image" src="https://user-images.githubusercontent.com/83772007/177552887-b692a8e6-b869-4f99-bd02-2f3a8faa0e34.png">

Also, consider adjusting the section title to "Support for manually deploying a custom scheduler has been removed."